src/SFA.DAS.Admin.Aan.Web/Controllers/ManageMembers/NetworkDirectoryController.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Aan.SharedUi.Constants;
+using SFA.DAS.Aan.SharedUi.Extensions;
+using SFA.DAS.Aan.SharedUi.Infrastructure;
+using SFA.DAS.Aan.SharedUi.Models;
+using SFA.DAS.Aan.SharedUi.Models.NetworkDirectory;
+using SFA.DAS.Aan.SharedUi.Models.NetworkEvents;
+using SFA.DAS.Aan.SharedUi.Services;
+using SFA.DAS.Admin.Aan.Application.OuterApi.Members;
+using SFA.DAS.Admin.Aan.Application.OuterApi.Regions;
+using SFA.DAS.Admin.Aan.Application.Services;
+using SFA.DAS.Admin.Aan.Web.Authentication;
+using SFA.DAS.ApprenticeAan.Web.Models.NetworkDirectory;
+
+namespace SFA.DAS.Admin.Aan.Web.Controllers.ManageMembers;
+
+[Authorize(Roles = Roles.ManageMembersRole)]
+[Route("network-directory", Name = SharedRouteNames.NetworkDirectory)]
+public class NetworkDirectoryController : Controller
+{
+    public const string RoleCheckListTitle = "Role";
+    public const string RoleCheckListQueryParameterName = "userRole";
+    public const string RegionCheckListTitle = "Region";
+    public const string RegionCheckListParameterName = "regionId";
+    private readonly IOuterApiClient _outerApiClient;
+
+    public NetworkDirectoryController(IOuterApiClient outerApiClient)
+    {
+        _outerApiClient = outerApiClient;
+    }
+
+    public async Task<IActionResult> Index(NetworkDirectoryRequestModel request, CancellationToken cancellationToken)
+    {
+        var getMembersTask = _outerApiClient.GetMembers(request.ToQueryStringParameters(), cancellationToken);
+        var regionTask = _outerApiClient.GetRegions(cancellationToken);
+
+        await Task.WhenAll(getMembersTask, regionTask);
+
+        var regions = regionTask.Result.Regions;
+        regions.Add(new Region(0, "Multi-regional", regions.Count + 1));
```

Comment:
bit fussy, and it's fine as is if you want to leave it, but I'd make the ordering a very high number (like 100), or based on the regions ordering maximum + 1

Reply:
It is static data and we know the numbers, so while I agree with you, this is ok for now.